tests/Load/config.sh

```
@@ -0,0 +1,66 @@
+INSTANCE_TAG=${INSTANCE_TAG:-$DEFAULT_INSTANCE_TAG}
+ROLE_NAME=${ROLE_NAME:-$DEFAULT_ROLE_NAME}
+BRANCH_NAME=${BRANCH_NAME:-$DEFAULT_BRANCH_NAME}
+SECURITY_GROUP_NAME=${SECURITY_GROUP_NAME:-$DEFAULT_SECURITY_GROUP_NAME}
+LOCAL_MODE=${LOCAL_MODE:-$DEFAULT_LOCAL_MODE}
+
+if [[ "$LOCAL_MODE" == "true" ]]; then
+    export ENDPOINT_URL=http://localhost:$LOCALSTACK_PORT
+    export AWS_ACCESS_KEY_ID=$AWS_SQS_KEY
+    export AWS_SECRET_ACCESS_KEY=$AWS_SQS_SECRET
+    AWS_CLI="aws --endpoint-url=${ENDPOINT_URL}"
+    STS_COMMAND="sts get-caller-identity --no-sign-request"
```

Shellcheck

[warning] STS_COMMAND appears unused. Verify use (or export if used externally).

(SC2034)